src/common/Throttle.cc

```diff
@@ -760,14 +814,13 @@ void TokenBucketThrottle::add_tokens() {
 }
 
 void TokenBucketThrottle::schedule_timer() {
-  add_tokens();
-
   m_token_ctx = new FunctionContext(
       [this](int r) {
         schedule_timer();
       });
+  m_timer->add_event_after(m_schedule_tick / 1000.0, m_token_ctx);
```

Comment:
add event every ms?

Reply:
`m_schedule_tick` is the filling period, and it is caculated by the `Bucket`.